.github/workflows/ci.yaml

```diff
@@ -98,6 +98,15 @@ jobs:
         path: dist/*.whl
         if-no-files-found: error
 
+    - name: Install python dependencies
+      run: ./dev.py install-python-deps
```

Comment:
This also installs our packages, and ideally we'd test the WHL files, because that's what end customers use. In rare cases, packages installed with `pip install -e .` may work, but packages installed by `pip install *.whl` will not work.

Reply:
Ok, I installed the wheel packages. Thanks